src/arrays.js

```diff
@@ -1,61 +1,67 @@
 const getNthElement = (index, array) => {
-  // your code here
+
+  return array[index % array.length]; 
 };
 
 const arrayToCSVString = array => {
-  // your code here
+  return array.join(',');
 };
 
 const csvStringToArray = string => {
-  // your code here
+ return string.split(',');
 };
 
 const addToArray = (element, array) => {
-  // your code here
+  array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+  return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+  array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+
+  return numbers.map(numbers => numbers.toString());
```

Comment:
```suggestion
  return numbers.map(number => number.toString());
```
The value declared in the callback is a placeholder value. As it isnt used outside of the callback its name is fairly arbitrary, but its good practice to NOT name it the same as the array you are mapping over